<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter your mobile IP</string>
   </property>
   <property name="placeholderText">
    <string>Ex: 192.168.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ex: 5555</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="connectBtn">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
